src/object.d

```diff
@@ -2186,6 +2192,21 @@ inout(V) get(K, V)(inout(V[K])* aa, K key, lazy inout(V) defaultValue)
     return (*aa).get(key, defaultValue);
 }
 
+unittest
```

Comment:
Why don't you simply make the unittest `@safe`?
`__compiles` is an anti-pattern and should only be used if absolutely necessary.